src/index.html

```diff
@@ -1,12 +1,332 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="../src/images/logo.png" alt="logo">
+      </a>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__item" data-qa="nav-hover">
+            <a href="#" class="header__link">
+              Laptops&nbsp;&&nbsp;computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
```

Comment:
You should check it everywhere
```suggestion
    </header>
    
    <main class="main">
```

Reply:
i use generel rules 
https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping